packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/GoogleMapBuilder.java

```diff
@@ -179,11 +177,6 @@ public void setInitialTileOverlays(List<Map<String, ?>> initialTileOverlays) {
     this.initialTileOverlays = initialTileOverlays;
   }
 
-  @Override
-  public void setMarkerCachingEnabled(boolean isEnabled) {
-    this.markerCachingEnabled = isEnabled;
-  }
```

Comment:
Just to better understand the change, in practical terms, what did this `markerCachingEnabled` works?

Reply:
When it renders a marker, if the marker was already rendered with all properties equal (label, color, etc), it will not re-draw the marker, will just fetch the bitmap from the cache. 

Actually i did not removed the cache, just removed the flag because we use it always activated